web/app.js

```diff
@@ -516,10 +516,15 @@ const PDFViewerApplication = {
     const container = appConfig.mainContainer,
       viewer = appConfig.viewerContainer;
     const annotationEditorMode = AppOptions.get("annotationEditorMode");
-    const pageColors = {
-      background: AppOptions.get("pageColorsBackground"),
-      foreground: AppOptions.get("pageColorsForeground"),
-    };
+    const pageColors = window.matchMedia("(forced-colors: active)").matches
```

Comment:
Generally speaking, it seems a little bit unfortunate to essentially make it impossible for users to specify *custom* values for these PDF.js-specific color preferences.

Looking at the bug, could we perhaps only disable this functionality when the "problematic" (general) Firefox preferences have been set?
*For context:* Years ago we had code which checked the `browser.display.document_color_use` preference; this was removed (since it'd become unused) in https://github.com/mozilla/pdf.js/pull/11521/commits/a977882f54c82cc7239e5238734194881a281137 and https://github.com/mozilla/gecko-dev/commit/7015ab6e166895bab2c6f8af30763b0b66f7b237

Would it make (any) sense to re-instate *similar* checks here, and do something like this instead?
```suggestion
    const pageColors =
      window.matchMedia("(forced-colors: active)").matches || this.supportsDocumentColors
```